CI/steam/scripts/install.bat

```diff
@@ -0,0 +1,85 @@
+@echo off
+@cd /d "%~dp0"
+
+goto checkAdmin
+
+
+:checkAdmin
+	net session >nul 2>&1
+	if %errorLevel% == 0 (
+		echo.
+	) else (
+		echo Administrative rights are required, please re-run this script as Administrator.
+		goto end
+	)
+
+:writeRegistry
+	reg add "HKLM\SOFTWARE\OBS Studio" /f /t REG_SZ /d %1 /reg:32
+	reg add "HKLM\SOFTWARE\OBS Studio" /f /t REG_SZ /d %1 /reg:64
+
+:setupProgramData
+	:: Required for UWP applications
+	mkdir "%PROGRAMDATA%\obs-studio-hook"
+	icacls "%PROGRAMDATA%\obs-studio-hook" /grant "ALL APPLICATION PACKAGES":(OI)(CI)(GR,GE)
+
+:checkDLL
+	echo Checking for 32-bit Virtual Cam registration...
+	reg query "HKLM\SOFTWARE\Classes\CLSID\{A3FCE0F5-3493-419F-958A-ABA1250EC20B}" >nul 2>&1 /reg:32
```

Comment:
Do we want to hard-code this in this file, or use a .bat.in file and grab this GUID from CI before producing the Steam build (which is what we do for other builds, I think)?

Reply:
Could be done, though I don't see us changing it nor anybody else needing this specific script to have their own.

Reply:
It would just be one less place to have to consider updating it should we change it, or if we want to register the Steam build and the normal build with different GUIDs to avoid clobbering each other or having one load the other (though that may also require us to differentiate the camera names).

As mentioned, that's how we currently handle this:
* https://github.com/obsproject/obs-studio/blob/ccd1ef2bd0abb7c9d86d25871dfd8744fdd450cb/plugins/win-dshow/virtualcam-install.bat.in
* https://github.com/obsproject/obs-studio/blob/ccd1ef2bd0abb7c9d86d25871dfd8744fdd450cb/plugins/win-dshow/virtualcam-uninstall.bat.in

I'm fine with this as-is for now, but something to consider.